config/v2_0/translate.go

```diff
@@ -1,25 +1,48 @@
 package v2_0
 
+import (
+	"fmt"
+
+	v1 "github.com/coreos/ignition/config/v1/types"
+	"github.com/coreos/ignition/config/v2_0/types"
+	"github.com/vincent-petithory/dataurl"
+)
+
+func intToPtr(x int) *int {
+	return &x
+}
+
+func strToPtr(s string) *string {
+	if s == "" {
+		return nil
+	}
+	return &s
+}
+
+func boolToPtr(b bool) *bool {
+	return &b
+}
+
 func TranslateFromV1(old v1.Config) types.Config {
 	config := types.Config{
 		Ignition: types.Ignition{
-			Version: v2_0.MaxVersion.String(),
+			Version: ConfigVersion,
```

Comment:
`ConfigVersion` I actually intended to remove, it's redundant with the already-existing `types.MaxVersion`. I'll remove it.

Ideally this will only use things that already exist. Is there anything else in this commit (or others) that jumps out at you as using things from the future?